ansible/roles/test/files/ptftests/arista.py

```diff
@@ -157,11 +158,11 @@ def run(self):
             log_output = self.do_cmd("show log | begin %s" % log_first_line)
             log_lines = log_output.split("\r\n")[1:-1]
             log_data = self.parse_logs(log_lines)
-            if len(log_data) != 0:
+            if len(log_data) > 1:
```

Comment:
why? magic constant

Reply:
In parse_logs(), we have this statement result['route_timeout'] = result_rt  before returning result when we don't find any bgp or portchannel info in the logs. log_data will have at least 1 key always.

Reply:
how do we know we have correct item when len(log_data) is bigger then 1?
Please check for particular keys.
Your check could stop working with another change in self.parse_logs()

Reply:
@neethajohn If

> log_data will have at least 1 key always

you may return a tuple, and the first element is `result_rt`

Reply:
improved the non empty check